src/test/test_subsumption.cpp

```
@@ -7,24 +7,27 @@
 
 using namespace prevail;
 
+using TypeRestrictions = std::vector<std::pair<Variable, TypeSet>>;
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)